content/en/docs/setup/upgrade/index.md

```diff
@@ -56,18 +56,24 @@ istio-sidecar-injector-canary   2020-04-28T19:03:26Z
 
 ### Data plane
 
-Simply installing the new revision has no impact on the existing proxies. To upgrade these,
-you must configure them to point to the new control plane. This is controlled during sidecar injection
+Installing the new revision has in place upgraded the istio-ingress gateway pod and service, which are automatically configured to point to the 'istiod-canary' control plane.
```

Comment:
Is this only the case for the istio-ingress gateway? What about egress gateways, or even other ingress-gateways that have been installed? I think this needs more explanation.

Reply:
cc @howardjohn

Reply:
All gateways are the same

Reply:
I only used istio-ingress gateway because that is the one enabled by default for default profile.  I'll update to include all gateways.

Reply:
reworded @frankbu PTAL and let me know